packages/intranet-header-workspace/projects/intranet-header/src/lib/swisspost-intranet-header.component.html

```diff
@@ -2,32 +2,40 @@
   <div class="nav-down">
     <div class="intranet-header">
       <div class="d-flex align-items-center flex-nowrap">
-        <div class="bg-logo d-sm-block">
+        <div id="logo" class="bg-logo d-sm-block">
```

Comment:
Why was this id added? There is a chance of this id already being in use by a project, which could cause hard to find errors. If this id is necessary, we should prefix it with `post-` or an even more specific prefix. Otherwise, please remove it.

*Same goes for the other instances where ids were added.*

Reply:
Yes, the IDs are needed to identify the containers and then calculate the available space. This identifies them uniquely. I see the point with the other projects and will add the prefix “post-”.